src/styles/main.scss

```diff
@@ -1,3 +1,68 @@
+@import './variables';
+
+* {
+  margin: 0;
+}
```

Comment:
it's not a good practice to reset margin like this. reset only for those tags you need